android/uhabits-android/src/main/res/menu/list_habits.xml

```diff
@@ -38,31 +51,31 @@
                 android:id="@+id/actionHideArchived"
                 android:checkable="true"
                 android:enabled="true"
-                android:title="@string/hide_archived"/>
```

Comment:
Please avoid proposing changes that only modify white space.

Reply:
Sorry this one slipped by me, and when reviewing my own code I couldn't be bothered to revert it. But if you want it I'll do it.